contributors/guide/github-workflow.md

```diff
@@ -235,3 +235,25 @@ git rebase -i upstream/master
 For mass automated fixups (e.g. automated doc formatting), use one or more
 commits for the changes to tooling and a final commit to apply the fixup en
 masse. This makes reviews easier.
+
+### Reverting a commit
+
+In case you wish to revert a commit, use the following instructions.
+
+_If you have upstream write access_, please refrain from using the
+`Revert` button in the GitHub UI for creating the PR, because GitHub
+will create the PR branch inside the main repository rather than inside your fork.
+
+```sh
+# create a branch
```

Comment:
Add a suggestion to start from a synced master branch?

Reply:
> Add a suggestion to start from a synced master branch?

Done.